nntrainer/compiler/ini_interpreter.cpp

```diff
@@ -275,12 +276,21 @@ void IniGraphInterpreter::serialize(
     IniSection s(layer->getName());
     s.setEntry("type", layer->getType());
 
-    /// @todo: implement export a property
-    std::cout << layer->getName() << std::endl;
+    Exporter e;
+    layer->export_to(e);
+
+    const auto key_val_pairs =
+      e.get_result<ExportMethods::METHOD_STRINGVECTOR>();
+
+    for (const auto &pair : key_val_pairs) {
+      s.setEntry(pair.first, pair.second);
+    }
+
+    sections.push_back(s);
```

Comment:
Just for a reminder, the order of sections is matters.

Reply:
I think activation should exist as a layer when we serialize (we have an assumption that it's been already trained). 
And I agree that time disributed layer should be considered later.